test/lib/deployment/test-deployment.js

```diff
@@ -1,6 +1,7 @@
 const assert = require('assert');
 const bufferReplace = require('buffer-replace');
 const fs = require('fs');
+const json5 = require('json5');
```

Comment:
Do you need to add json5 in the root package.json?

Reply:
Added 👍